tools/pipelines/repo-policy-check.yml

```diff
@@ -15,52 +15,80 @@ pr:
 - lts
 - release/*
 
-pool:
-  Small
-
 variables:
 - name: skipComponentGovernanceDetection
   value: true
 - name: pnpmStorePath
   value: $(Pipeline.Workspace)/.pnpm-store
 
-steps:
-- template: templates/include-use-node-version.yml
-
-- template: templates/include-install-pnpm.yml
+# The `resources` specify the location and version of the 1ES PT.
+resources:
+  repositories:
+  - repository: m365Pipelines
+    type: git
+    name: 1ESPipelineTemplates/M365GPT
+    ref: refs/tags/release
+extends:
+  # The pipeline extends the 1ES PT which will inject different SDL and compliance tasks.
+  template: v1/M365.Official.PipelineTemplate.yml@m365Pipelines
   parameters:
-    buildDirectory: .
+    pool:
+      name: Small-1ES # This is one of the Fluid Orgs new 1ES hosted pools.
+      os: linux
+    sdl:
+      arrow:
+        # This is the service connection for the Arrow Service Connection in FluidFramework Azure DevOps organization
+        # Currently we want to use different names for internal and public builds for Arrow Service Connection
+        ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+          serviceConnection: Arrow_FluidFramework_internal
+        ${{ else }}:
+          serviceConnection: Arrow_FluidFramework_public
+      sourceAnalysisPool:
+        name: Azure-Pipelines-1ESPT-ExDShared
+        image: windows-2022
+        os: windows
```

Comment:
This seems to be missing the setting to allow PRs from forks to succeed. @seanimam can you close this PR and start it from a branch in your fork? We should make sure that flow works correctly.